Doc/glossary.rst

```diff
@@ -1033,17 +1037,40 @@ Glossary
       :attr:`~instance.__class__` attribute or can be retrieved with
       ``type(obj)``.
 
+   type alias
+      A synonym for a type, created by assigning the type to an identifier.
+
+      Type aliases are useful for simplifying :term:`type hints <type hint>`.
+      For example ::
+
+         from typing import List, Tuple
+
+         Color = Tuple[int, int, int]
+
+         def remove_gray_shades(colors: List[Color]) -> List[Color]:
+             pass
+
+      is equivalent to ::
+
+         def remove_gray_shades(
+                 colors: List[Tuple[int, int, int]]) -> List[Tuple[int, int, int]]:
+             pass
+
+      See :mod:`typing` and :pep:`484`, which describe this functionality.
+
    type hint
-      A specification about the expected type for a global variable, class
-      variable, function or method parameter or return value.
+      An :term:`annotation` that specifies the expected type for a variable, class
+      attribute, function or method parameter or return value.
 
-      While type hints are optional and are not enforced by Python when used,
-      they are useful for static type analysis tools, and aid IDEs on code
+      Type hints are optional and are not enforced by Python but
+      they are useful to static type analysis tools, and aid IDEs on code
       completion and refactoring.
 
-      Type hints are stored in :term:`annotations <annotation>`.
+      Type hints of global variables, class attributes, functions and methods
+      parameters and return values, but not local variables, can be accessed using
+      :func:`typing.get_type_hints`.
 
-      See also :pep:`483` which describe this functionality.
+      See :mod:`typing` and :pep:`483`, which describe this functionality.
```

Comment:
I think this should be PEP 484. (PEP 483 also describes type hints but is much less precise and rarely referenced.)